src/types/Filter.ts

```diff
@@ -0,0 +1,5 @@
+export enum FilteringOption {
+  all = 'All',
+  active = 'Active',
+  completed = 'Completed',
```

Comment:
nice point! The only thing is that I had such a variant from the very beginning, but was asked to remake that. So that now I have my enum in lowercase.
<img width="444" alt="image" src="https://github.com/mate-academy/react_todo-app-with-api/assets/121095730/4cfc882c-6a25-4664-b91e-3a8476dcd090">